game/settings_Panel.gd

```diff
@@ -94,13 +106,6 @@ func _on_right_sable_col_color_changed(color,overwrite=true):
 
 
 func _on_saber_tail_toggled(button_pressed,overwrite=true):
-	if button_pressed:
-		game.left_saber.set_tail_size(3)
-		game.right_saber.set_tail_size(3)
-	else:
-		game.left_saber.set_tail_size(0)
-		game.right_saber.set_tail_size(0)
-		
```

Comment:
The "Saber Tail" isn't togglable anymore. Remember to reimplement this (probably using a bool in default_saber :)